styles.css

```diff
@@ -373,3 +373,234 @@ body {
 .section3 > .about > p {
   padding: 20;
 }
+
+.section4 {
+  background-color: white;
+  width: 100%;
+  overflow: hidden;
+}
+
+.contact-us {
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  border-top-left-radius: 7rem;
+  display: flex;
+  flex-direction: column;
+  padding-left: 0;
+  padding-left: 10px;
+  background-size: cover;
+  overflow: hidden;
+}
+
+.background-image {
+  background-image: url("../image/contact-background.svg");
+  background-repeat: no-repeat;
+  background-position: 139px 9px;
+  background-size: 305px 782px;
+}
+
+.contact-header > .c-header {
+  font-size: 40px;
+  font-weight: bold;
+  padding-top: 100px;
+  color: white;
+  text-align: center;
+  margin: 0;
+}
+
+.contact-header > .description > .c-descrp {
+  color: white;
+  text-align: center;
+  font-size: 20px;
+  padding: 20px;
+}
+
+.contact-us img {
+  padding-top: 15px;
+  max-width: 100%;
+  position: absolute;
+  padding-left: 140px;
+}
+
+form {
+  z-index: 19;
+  padding: 5%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+form input {
+  height: 48px;
+  border-radius: 8px;
+  padding: 0 10px;
+  border: none;
+  margin-bottom: 20px;
+}
+
+form textarea {
+  height: 170px;
+  border-radius: 8px;
+  padding: 10px 10px;
+  border: none;
+  margin-bottom: 20px;
+}
+
+form input[type="submit"] {
+  height: 48px;
+  width: 140px;
+  border-radius: 8px;
+  padding: 0 10px;
+  border: none;
+  color: #6070ff;
+  margin-bottom: 20px;
+}
+
+form input[type="submit"]:hover {
+  height: 48px;
+  width: 140px;
+  border-radius: 8px;
+  padding: 0 10px;
+  border: 1px solid white;
+  color: white;
+  background: #6070ff;
+  margin-bottom: 20px;
+}
+
+.dash {
+  border-bottom: 6px solid #0d151e;
+  margin: 0 35%;
+  border-radius: 10px;
+}
+
+::-webkit-scrollbar {
+  width: 10px;
+  height: 10px;
+  display: none;
+}
+
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 5px grey;
+  border-radius: 10px;
+}
+
+::-webkit-scrollbar-thumb {
+  background: #6070ff;
+  border-radius: 10px;
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background: #3847d4;
+}
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 780px;
+  padding: 120px 24px;
+  font-family: Poppins, system-ui, apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  font-style: normal;
+  background-image: url("./image/contact\ form\ shapes-mobile@2x.png");
+  background-color: #6070ff;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: bottom -30px right -16px;
+}
+
+.paragraph {
+  font-family: 'Poppins', sans-serif;
+  color: #ebebff;
+  font-style: normal;
+  font-size: 20px;
+  line-height: 28px;
+  vertical-align: top;
+  width: 327px;
+  height: 102px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```

Comment:
- [ ] You did a great job in the design of the contact form but there is a small issue in your design that you need to fix. the text of the paragraph must be aligned in the middle of your page kindly fix that it will make your design stick to the Figma design as required in the project, to do that I just suggest you add in this the style of the paragraph  `   text-align: center;` :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/206708747-8bd1c940-3742-46e1-907e-4ac64f592195.png)

## Figma design

![image](https://user-images.githubusercontent.com/95297251/206708976-245a5de0-7dd9-49bc-a596-1514ea9356ad.png)